Source/WTF/wtf/NativePromise.h

```diff
@@ -1033,11 +1037,11 @@ class NativePromise<ResolveValueT, RejectValueT, options>::Producer final : publ
         assertIsDead();
     }
 
-    explicit operator bool() const { return m_promise && m_promise->isResolvedOrRejected(); }
+    explicit operator bool() const { return m_promise && m_promise->isSettled(); }
     bool isNothing() const
```

Comment:
s/isNothing/isNotSettled/

Reply:
I'm going to leave that for now, there's too much use of it and it's purely for assertion purposes